### PR TITLE
AST-490 - fix:  Removed default border bottom for transparent header and make it as 0 value.

### PR DESCRIPTION
**Pull request by sagarb90**
### Description
- https://wp-astra.atlassian.net/browse/AST-490
- Removed default border bottom for transparent header and make it as 0 value as previous was blank so it get applied the normal header border.

### Screenshots
https://share.getcloudapp.com/rRuj6Z1Y

### Types of changes
Bug fix (non-breaking change which fixes an issue) .

### Checklist:
- [x] My code is tested
- [x] My code passes the PHPCS tests
- [x] My code follows accessibility standards 
- [ ] My code has proper inline documentation 
- [ ] I've included any necessary tests 
- [ ] I've included developer documentation 
- [ ] I've added proper labels to this pull request

### CODE REVIEW

**Review by premanshup — APPROVED** [review]

**Review by abhishek-bsf — APPROVED** [review]
@sagarb90  Please add changelog if required.

File: inc/extras.php [hunk 1]
```diff
@@ -585,3 +585,14 @@ function astra_target_rules_for_related_posts() {
 function is_astra_addon_3_5_0_version() {
 	return defined( 'ASTRA_EXT_VER' ) && version_compare( ASTRA_EXT_VER, '3.5.0', '<' );
 }
+
+/**
+ * Set flag to manage backward compatibility for v3.5.0 earlier users for the transparent header border bottom default value changed.
+ *
+ * @since  x.x.x
+ */
+function apply_updated_default_tranparent_header_value() {
```
Comment (premanshup) [inline inc/extras.php:594]:
```suggestion
function astra_transparent_header_default_value() {
```

**Review by premanshup — CHANGES_REQUESTED** [review]

File: inc/extras.php (hunk 1 above)
Comment (sagarb90) [reply inc/extras.php]: This is done rename funciton name

File: changelog.txt [hunk 2]
```diff
@@ -8,6 +8,7 @@ v3.6.0 (Unreleased)
 - Improvement: Changed customizer submenu control title from 'Container Animation' to 'Submenu Animation'.
 - Improvement: Moved 'Submenu Animation' control from General tab to Design  tab.
 - Improvement: Changed sub-menu container animation option name from 'Default' to 'None'.
+- Improvements: Changed Transparent Header Border Bottom default value from '' to 0;
```
Comment (premanshup) [inline changelog.txt:11]:
```suggestion
- Improvement: Changed Transparent Header Border Bottom default value from '' to 0;
```

Comment (sagarb90) [reply changelog.txt]: Done

**Review by premanshup — DISMISSED** [review]

File: changelog.txt [hunk 3]
```diff
@@ -10,6 +10,10 @@ v3.6.0 (Unreleased)
 - Fix: Secondary menu not expendable on mobile when it is outside of Off Canvas.
 - Fix: CLS – Flashing desktop menu on mobile view.
 - Fix: WooCommerce - product search widget button not aligned on other than shop and single product page.
+- Improvement: Changed customizer submenu control title from 'Container Animation' to 'Submenu Animation'.
+- Improvement: Moved 'Submenu Animation' control from General tab to Design  tab.
+- Improvement: Changed sub-menu container animation option name from 'Default' to 'None'.
```
Comment (imnavanath) [inline changelog.txt:15]: @sagarb90 all these 3 points are already covered in the above changelog. L2-5

Comment (sagarb90) [reply changelog.txt]: @imnavanath Yes Done, thanks.

**Review by imnavanath — APPROVED** [review]

**Pull request merged**